src/components/TradeWidget/OrderValidity.tsx

```diff
@@ -227,7 +227,7 @@ const OrderValidityBox = styled(PriceInputBox)`
 `
 
 const OrderStartsTooltip = (
-  <HelpTooltipContainer>Orders that are valid ASAP will be considered for the next batch.</HelpTooltipContainer>
+  <HelpTooltipContainer>Orders that are valid now will be considered for the next batch.</HelpTooltipContainer>
```

Comment:
Capitalize NOW?

Reply:
mm idk @alfetopito @anxolin

Reply:
I would make it bold or highlighted somehow, not capitalized.

Reply:
![image](https://user-images.githubusercontent.com/2352112/87065053-d3637f80-c210-11ea-94c0-051156b04224.png)

Yes, probably good to capitalize and make it 💪strong. Since it's just describing what **Now** means in this context.

Is it worth it to mention you can changing it by clicking the GEAR icon...?